Themes/default/Post.template.php

```diff
@@ -362,7 +362,7 @@ function addPollOption()
 							<div class="files" id="attachment_previews">
 								<div id="au-template">
 									<div class="attachment_preview_wrapper">
-										<div class="attach-ui">
+										<div class="attach-ui roundframe">
```

Comment:
why is an ui change needed for this issue?

Reply:
It's not necessary. Maybe it's because I haven't updated the fork in a while.

Reply:
I suspect that part of the recent attachment UI changes were merged into this PR.